pydis_site/templates/events/pages/code-jams/8/github-bootcamp.html

```diff
@@ -13,24 +13,29 @@
 
 {% block event_content %}
     <p>
-        This year we'll be running a GitHub Bootcamp on Saturday, June 26th.
+        <strong>This year we'll be running a GitHub Bootcamp on Saturday, June 26th at 4PM UTC.</strong>
         This bootcamp is intended to be an interactive workshop session where we help participants
-        get setup with git and using it in a team environment.
+        get setup with git and using it in a team environment. If you are new to git or would like to brush up on it,
+        then this is the perfect event for you.
+    </p>
+    <p>
+        The instructional parts of this bootcamp will be recorded. We'll also be providing a cheatsheet / reference guide to all attendees that will
```

Comment:
```suggestion
        The instructional parts of this Bootcamp will be recorded. We'll also be providing a cheatsheet/reference guide to all attendees that will
```
In the first sentence with `strong`, `bootcamp` is capital.